tools/cldr-code/src/main/java/org/unicode/cldr/util/CaseIterator.java

```diff
@@ -48,164 +49,593 @@ public class CaseIterator {
 
     private static String[][] exceptionList = {
         // a\N{MODIFIER LETTER RIGHT HALF RING}
-        { "a\u02BE", "A\u02BE", "a\u02BE", },
+        { "a\u02BE", "A\u02BE", "a\u02BE" },
         // ff
-        { "ff", "FF", "Ff", "fF", "ff", },
+        { "ff", "FF", "Ff", "fF", "ff" },
         // ffi
-        { "ffi", "FFI", "FFi", "FfI", "Ffi", "F\uFB01", "fFI", "fFi", "ffI", "ffi", "f\uFB01", "\uFB00I", "\uFB00i", },
+        {
+            "ffi",
```

Comment:
Another bad case. No advantage at all in splitting all these up; just wastes vertical lines.